web/app/scripts/filterbar/date-range-directive.js

```diff
@@ -0,0 +1,68 @@
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function DateRangeField(FilterState, $timeout) {
+        var module = {
+            restrict: 'A',
+            require: ['^driver-filterbar', 'date-range-field'],
+            templateUrl: 'scripts/filterbar/date-range.html',
+            controller: 'dateRangeController',
+            link: function(scope, elem, attrs, ctlArray) {
+                var filterLabel = '__dateRange';
+                init();
+                scope.calendarOptions = {'format': ctlArray[1].dateTimeFormat};
```

Comment:
It would be nice to locally define controller vars instead of referencing them by array offset, for clarity.